### PR TITLE
Make config fields static

### PR DESCRIPTION
**Pull request by agavrilov76**

### CODE REVIEW

File: core/src/main/java/com/github/secondbase/core/SecondBase.java [hunk 1]
```diff
@@ -72,4 +76,20 @@ public SecondBase(
     public Flags getFlags() {
         return flags;
     }
+
+
+    /**
```
Comment (neumayer) [inline core/src/main/java/com/github/secondbase/core/SecondBase.java:81]: The comment is copy-pasted from the interface?

File: example/src/main/java/com/github/secondbase/example/main/HelloAll.java [hunk 2]
```diff
@@ -99,8 +101,13 @@ public static void main(final String[] args) throws SecondBaseException, IOExcep
 
         final Flags flags = new Flags().loadOpts(HelloAll.class);
 
-        new SecondBase(realArgs, modules, flags);
+        final SecondBase secondBase = new SecondBase(realArgs, modules, flags);
+
+        final HttpServer server = startHelloAllService();
+
+        Thread.sleep(1000);
```
Comment (neumayer) [inline example/src/main/java/com/github/secondbase/example/main/HelloAll.java:108]: Have a comment here? And remove some of the empty lines?

**Review by neumayer — COMMENTED** [review]
There's quite a bit happening here and I wonder if it could be split up maybe? Also doesn't it do the opposite of what the title says, i.e. make stuff "non-static?"

**Comment by agavrilov76** [comment]
@neumayer I'll split it and make it nicer.
The rationale for all this refactoring is to introduce an integration test library which provides a JUnit rule that would start a service  before the test case and stop it afterwards.

**Comment by neumayer** [comment]
Ok, that sounds nice indeed.

**Pull request closed without merging**